sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelector.java

```diff
@@ -180,7 +180,7 @@ public SettingSelector setFields(SettingFields... fields) {
     }
 
     /**
-     * Gets the {@link Range} used to select a specific range of revisions with {@code listSettingRevisions}. If
+     * Gets the {@link Range} used to select a specific range of revisions with {@code listRevisions}. If
```

Comment:
I would add a `@see ConfigurationAsyncClient#listRevisions` and `@see ConfigurationClient#listRevisions` to this method, so even though the Javadoc isn't linked, they have a pointer to it.